index.d.ts

```diff
@@ -602,4 +764,4 @@ openSystemPreferences('security', 'Firewall');
 @param section - The section within that pane.
 @returns A Promise that resolves when the preferences window is opened.
 */
-export const openSystemPreferences: <T extends keyof SystemPreferencesPanes>(pane?: T, section?: SystemPreferencesPanes[T]) => Promise<void>;
+export const openSystemPreferences: <T extends keyof SystemPreferencesMacOsPanes>(pane?: T|SystemPreferencesWindowsPane, section?: SystemPreferencesMacOsPanes[T]) => Promise<void>;
```

Comment:
I think it would be better to use an overload here.

Reply:
Sorry, been quite busy past days. Just pushed last commit!